apis/v1alpha2/tcproute_types.go

```diff
@@ -93,7 +93,9 @@ type TCPRouteRule struct {
 	Matches []TCPRouteMatch `json:"matches,omitempty"`
 
 	// BackendRefs defines the backend(s) where matching requests should be
-	// sent.
+	// sent. If unspecified or invalid, the rule performs no forwarding; if
+	// no filters are specified that would result in a response being sent,
+	// a HTTP 503 status code is returned.
```

Comment:
Missed this the first time, but the HTTP 503 bit doesn't make sense for TCP or other L4 protocols.

Reply:
Close the connection for TCP and ignore the packet for UDP?

Reply:
Maybe just "the rule must not be configured in the underlying implementation"?